Answers.md

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Very solid answers Stefan 👍 I like the balance you're striking between technical and non-technical language. I had to google the 🐡 cipher, very cool to read about that!